lib/kitchen_sink/boolean.ex

```diff
@@ -0,0 +1,35 @@
+defmodule KitchenSink.Boolean do
+
+  @moduledoc """
+  this module is for Boolean functions
+  """
+
+  defp string_bool?("true"), do: true
+  defp string_bool?("false"), do: false
+  defp string_bool?("t"), do: true
+  defp string_bool?("f"), do: false
+  defp string_bool?("on"), do: true
+  defp string_bool?("off"), do: false
+  defp string_bool?("yes"), do: true
+  defp string_bool?("no"), do: false
+  defp string_bool?("y"), do: true
+  defp string_bool?("n"), do: false
+  defp string_bool?(_), do: :error
```

Comment:
how do i modify the idea of a boolean?
what if some japanese dude wants to use this and 'h' is their idea of true